src/cljs/sixsq/slipstream/webui/main/events.cljs

```diff
@@ -8,11 +8,26 @@
     [taoensso.timbre :as log]))
 
 
+(reg-event-db
+  ::set-device
+  (fn [db [_ device]]
+    (log/error "setting device:" device)
```

Comment:
A bit strong for an info message. It's all okey for me to merge

Reply:
Indeed.  Debugging statement leaking in.  Fixed.